backend/apps/web/models/auths.py

```diff
@@ -12,15 +13,23 @@
     create_token,
 )
 
-import config
-
-DB = config.DB
+from apps.web.internal.db import DB
 
 ####################
 # DB MODEL
 ####################
 
 
+class Auth(Model):
+    id = CharField(unique=True)
```

Comment:
question: we're saving uuid in the `id` columns that's why its CharField?

Reply:
Yep, basically uuid casted as a string!